app/assets/javascripts/application.js

```diff
@@ -1,2 +1,4 @@
 //= require turbolinks
-//= require app/utils
+
+import 'app/utils';
+import 'app/pw-toggle';
```

Comment:
the browserify build step wasn't working properly when files were included the "rails" way; switched to the "javascript" way (it is JS after all 😄)